Purchases/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -27,16 +27,16 @@ class PurchasesOrchestrator {
     var finishTransactions: Bool { systemInfo.finishTransactions }
     var allowSharingAppStoreAccount: Bool {
         get {
-            return maybeAllowSharingAppStoreAccount ?? identityManager.currentUserIsAnonymous
+            return _allowSharingAppStoreAccount ?? identityManager.currentUserIsAnonymous
```

Comment:
I was really not sure what to go with for this one, open to ideas!

Reply:
This matches what Apple typically does.